tests/system-ovn.at

```diff
@@ -10660,3 +10660,261 @@ OVS_TRAFFIC_VSWITCHD_STOP(["/failed to query port patch-.*/d
 /connection dropped.*/d"])
 AT_CLEANUP
 ])
+
+OVN_FOR_EACH_NORTHD([
+AT_SETUP([1 LR to test ARP proxy convinations])
+AT_KEYWORDS([proxy-arp])
+
+CHECK_CONNTRACK()
+CHECK_CONNTRACK_NAT()
```

Comment:
Doesn't seem like the is doing something wth NAT and CT so those two checks are not needed. However there is a missing check for tcpdump. Also it is usually good to put the topology to comment somewhere near top.

Reply:
Which tcpdump is missing ?

Reply:
`AT_SKIP_IF([test $HAVE_TCPDUMP = no])`